app/portainer/views/endpoints/edit/endpoint.html

```diff
@@ -211,6 +211,46 @@
             <por-endpoint-security form-data="formValues.SecurityFormData" endpoint="endpoint"></por-endpoint-security>
           </div>
           <!-- !endpoint-security -->
+          <!-- open-amt info -->
+          <div ng-if="state.showAMTInfo">
+            <div class="col-sm-12 form-section-title">
+              Open Active Management Technology
+            </div>
+            <div class="form-group">
+              <label for="endpoint_managementinfo" class="col-sm-3 col-lg-2 control-label text-left">
+                Raw Information
```

Comment:
Can we rename this and/or break it down into meaningful information?

Reply:
For that we need Intel's change so receive a JSON, I believe this this a placeholder label for at least showing the information back from the `amtinfo` command

Reply:
Ok, that makes sense.